aw_datastore/storages/abstract.py

```diff
@@ -26,7 +26,7 @@ def create_bucket(self, bucket_id: str, type_id: str, client: str,
         raise NotImplementedError
 
     @abstractmethod
-    def delete_bucket(self, bucket_id: str) -> None:
+    def delete_bucket(self, bucket_id: str) -> bool:
```

Comment:
If delete_bucket fails it should not return false, it should raise an exception.
If you get a failure from delete_bucket you also want to know the cause.

Reply:
Fixed